README.md

```diff
@@ -8,11 +8,13 @@ Implements the [EDN format](https://github.com/edn-format/edn) in Python.
 
 ## Usage ##
 
-    >>> import edn_format
-    >>> edn_format.dumps({1, 2, 3})
-    '#{1 2 3}'
-    >>> edn_format.loads("[1 true nil]")
-    [1, True, None]
+```pycon
```

Comment:
No, `python` is for Python code. Those `>>>` would confuse the parser; thankfully there’s a parser for Python’s console that handles them nicely.